app/views/welcome/index.html.erb

```diff
@@ -1,7 +1,6 @@
 <div id="title">
   <h1>Areas where you can help!</h1>
 
-  <%= render 'shared/category_row' %>
```

Comment:
The front page has categories and a categories bar? Redundant and I think it looks bad. We should talk about wireframes as a group IMO in the near future because we're writing tests for things that look bad and won't exist in the near future which is time/effort wasted.